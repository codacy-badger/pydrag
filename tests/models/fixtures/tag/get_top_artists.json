{
    "version": 1,
    "interactions": [
        {
            "request": {
                "method": "GET",
                "uri": "https://ws.audioscrobbler.com/2.0/?api_key=LAST_FM_API_KEY&format=json&limit=2&method=tag.getTopArtists&page=1&tag=rap",
                "body": null,
                "headers": {
                    "User-Agent": [
                        "python-requests/2.20.0"
                    ],
                    "Accept-Encoding": [
                        "gzip, deflate"
                    ],
                    "Accept": [
                        "*/*"
                    ],
                    "Connection": [
                        "keep-alive"
                    ]
                }
            },
            "response": {
                "status": {
                    "code": 200,
                    "message": "OK"
                },
                "headers": {
                    "Server": [
                        "openresty/1.9.7.5"
                    ],
                    "Access-Control-Max-Age": [
                        "86400"
                    ],
                    "Connection": [
                        "keep-alive"
                    ],
                    "Date": [
                        "Sat, 10 Nov 2018 11:27:55 GMT"
                    ],
                    "Access-Control-Allow-Origin": [
                        "*"
                    ],
                    "Access-Control-Request-Headers": [
                        "Origin, X-Atmosphere-tracking-id, X-Atmosphere-Framework, X-Cache-Date, Content-Type, X-Atmosphere-Transport, *"
                    ],
                    "Content-Type": [
                        "application/json; charset=UTF-8"
                    ],
                    "Access-Control-Allow-Methods": [
                        "POST, GET, OPTIONS"
                    ],
                    "Content-Length": [
                        "1469"
                    ]
                },
                "body": {
                    "string": "{\"topartists\":{\"artist\":[{\"name\":\"Eminem\",\"mbid\":\"CENSORED13d4\",\"url\":\"https://www.last.fm/music/Eminem\",\"streamable\":\"0\",\"image\":[{\"#text\":\"https://lastfm-img2.akamaized.net/i/u/34s/CENSORED.png\",\"size\":\"small\"},{\"#text\":\"https://lastfm-img2.akamaized.net/i/u/64s/CENSORED.png\",\"size\":\"medium\"},{\"#text\":\"https://lastfm-img2.akamaized.net/i/u/174s/CENSORED.png\",\"size\":\"large\"},{\"#text\":\"https://lastfm-img2.akamaized.net/i/u/300x300/CENSORED.png\",\"size\":\"extralarge\"},{\"#text\":\"https://lastfm-img2.akamaized.net/i/u/300x300/CENSORED.png\",\"size\":\"mega\"}],\"@attr\":{\"rank\":\"1\"}},{\"name\":\"Lil' Wayne\",\"url\":\"https://www.last.fm/music/Lil%27+Wayne\",\"streamable\":\"0\",\"image\":[{\"#text\":\"https://lastfm-img2.akamaized.net/i/u/34s/CENSORED.png\",\"size\":\"small\"},{\"#text\":\"https://lastfm-img2.akamaized.net/i/u/64s/CENSORED.png\",\"size\":\"medium\"},{\"#text\":\"https://lastfm-img2.akamaized.net/i/u/174s/CENSORED.png\",\"size\":\"large\"},{\"#text\":\"https://lastfm-img2.akamaized.net/i/u/300x300/CENSORED.png\",\"size\":\"extralarge\"},{\"#text\":\"https://lastfm-img2.akamaized.net/i/u/300x300/CENSORED.png\",\"size\":\"mega\"}],\"@attr\":{\"rank\":\"2\"}}],\"@attr\":{\"tag\":\"rap\",\"page\":\"1\",\"perPage\":\"2\",\"totalPages\":\"30763\",\"total\":\"61526\"}}}"
                }
            }
        }
    ]
}
