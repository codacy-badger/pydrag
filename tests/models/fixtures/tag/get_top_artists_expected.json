{
    "data": [
        {
            "attr": {
                "rank": 1
            },
            "image": [
                {
                    "size": "small",
                    "text": "https://lastfm-img2.akamaized.net/i/u/34s/CENSORED.png"
                },
                {
                    "size": "medium",
                    "text": "https://lastfm-img2.akamaized.net/i/u/64s/CENSORED.png"
                },
                {
                    "size": "large",
                    "text": "https://lastfm-img2.akamaized.net/i/u/174s/CENSORED.png"
                },
                {
                    "size": "extralarge",
                    "text": "https://lastfm-img2.akamaized.net/i/u/300x300/CENSORED.png"
                },
                {
                    "size": "mega",
                    "text": "https://lastfm-img2.akamaized.net/i/u/300x300/CENSORED.png"
                }
            ],
            "mbid": "CENSORED13d4",
            "name": "Eminem",
            "url": "https://www.last.fm/music/Eminem"
        },
        {
            "attr": {
                "rank": 2
            },
            "image": [
                {
                    "size": "small",
                    "text": "https://lastfm-img2.akamaized.net/i/u/34s/CENSORED.png"
                },
                {
                    "size": "medium",
                    "text": "https://lastfm-img2.akamaized.net/i/u/64s/CENSORED.png"
                },
                {
                    "size": "large",
                    "text": "https://lastfm-img2.akamaized.net/i/u/174s/CENSORED.png"
                },
                {
                    "size": "extralarge",
                    "text": "https://lastfm-img2.akamaized.net/i/u/300x300/CENSORED.png"
                },
                {
                    "size": "mega",
                    "text": "https://lastfm-img2.akamaized.net/i/u/300x300/CENSORED.png"
                }
            ],
            "name": "Lil' Wayne",
            "url": "https://www.last.fm/music/Lil%27+Wayne"
        }
    ]
}
